icons/inspector.svg

```diff
@@ -0,0 +1,17 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="14" height="14" x="5" y="5" rx="2" />
+  <path d="M12 5V2" />
+  <path d="M19 12h3" />
+  <path d="M12 22v-3" />
+  <path d="M2 12h3" />
```

Comment:
What about just calling it `scope` and adding a dot in the middle?

![icon](https://github.com/lucide-icons/lucide/assets/25524993/82060a77-bfb7-40bf-b0d1-8b47521bc30b)

```suggestion
  <path d="M2 12h3" />
  <path d="M12 12h0.01" />
```

Reply:
> What about just calling it `scope` and adding a dot in the middle?

Already something similar in #1338.